docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+# DALi Widget Application
+
+
+You can create a widget applications with DALi to display small version of application on the homescreen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> A DALi widget application is implemented similarly to a [DALi basic UI application](dali-basic-app.md), since the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class. Consequently, make sure you are familiar with the basic UI application details before tackling a widget application.
+
+- Make sure you [start the event loop](#mainloop).
+
+- Implement your own widget class by inheriting the [Dali::Widget class](#widget-api).
+
+
+**Figure: DALi widget application**
+
+![DALi widget application](./media/dali_widget_application.png)
+
+For the steps of creating the basic parts of a simple DALi widget application, see [Basics of Creating a DALi Widget Application](#create).
+
+<a name="widget-application-api"></a>
+## Widget application class
+
+If you want to use DALi for your widget application, you must create the application with the `Dali::WidgetApplication` class. This class provides the means for initializing the resources required by DALi.
+
+The following table lists the main function you need in your widget application.
+
+**Table: Main functions**
+
+| Function                             | Description                              |
+|--------------------------------------|------------------------------------------|
+| `mainloop()`                         |
+| `RegisterWidgetCreatingFunction()`   | Register function to create widget instance |
+
+The `mainloop()` function is used to start the event loop. If you do not call the function and start the event loop, DALi cannot call any callback functions for application events.
+
+The `RegisterWidgetCreatingFunction()` function registers creating function to create widget instance. Widget application will create widget instance using this creating function when WidgetViewer application requests.
```

Comment:
The `RegisterWidgetCreatingFunction()` function registers creating function to create widget instance. Widget application will create widget instance using this creating function when WidgetViewer application requests.

Delete this sentence, since it is added as a part of table "Main functions".